lib/perilune/configuration.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Perilune
+  class Configuration
+    attr_accessor :queue_name
+
+    def initialize
+      @queue_name = queue_name || 'default'
```

Comment:
Whats the use of `queue_name`  here? It is not passed into `initialize` method, so it just try to read it from accessor. Which during initialization is gonna be `nil`.
```suggestion
      @queue_name = :default
```